### PR TITLE
chore: create separate webpack

### PR DESCRIPTION
**Pull request by JeevaRamu0104**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description
Current webpack build is building with public folder which contains the static assets.Added new webpack configuration which ignores copying the static assets 

## How did you test it?
All the icons should appear even after doing this deployment

## Where to test it?

- [ ] INTEG
- [ ] SANDBOX
- [ ] PROD

## Checklist

- [ ] I ran `npm run re:build`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: public/hyperswitch/index.html [hunk 1]
```diff
@@ -98,7 +98,7 @@
 
 <body>
   <!-- SDK Loader Manager (noscript) -->
-  <script defer type="module" src="/wasm/euclid.js"></script>
+  <!-- <script defer type="module" src="/wasm/euclid.js"></script> -->
```
Comment (gitanjli525) [inline public/hyperswitch/index.html:101]: why is this commented out ?

Comment (JeevaRamu0104) [reply public/hyperswitch/index.html]: updated

**Review by gitanjli525 — DISMISSED** [review]

File: config/config.toml [hunk 2]
```diff
@@ -3,7 +3,7 @@ primary_color="#006DF9"
 primary_hover_color="#005ED6"
 sidebar_color="#242F48"
 [default.endpoints]
-api_url="http://localhost:8080"
+api_url="https://sandbox.hyperswitch.io"
```
Comment (Riddhiagrawal001) [inline config/config.toml:6]: Please revert the changes in this file

**Review by Riddhiagrawal001 — DISMISSED** [review]

**Comment by Riddhiagrawal001** [comment]
LGTM

File: webpack.custom.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
+const ReactRefreshWebpackPlugin = require("@pmmmwh/react-refresh-webpack-plugin");
```
Comment (Riddhiagrawal001) [inline webpack.custom.js:1]: why this plugin is only added to custom.js and not other webpack configs?

**Review by sagarnaikjuspay — APPROVED** [review]

File: config/config.toml (hunk 2 above)
Comment (JeevaRamu0104) [reply config/config.toml]: updated

**Review by gitanjli525 — APPROVED** [review]

**Review by kanikabansal-juspay — APPROVED** [review]
